code/index.html

```diff
@@ -1,22 +1,153 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Lion News</title>
+  <link rel="stylesheet" href="./style.css" />
+  <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body>
+
+  <!-- HEADER WITH LOGO NAME AND NAV BAR -->
+  <header>
+
+    <h3>LION</h3>
+
+    <img src="Lejon.png" alt="Lion logo">
+
+    <h1>LION TIMES</h1>
+
+    <h3>TIMES</h3>
+
+    <nav>
+      <a href="https://www.africaguide.com/wildlife/lion.htm" target="_blank" title="Africa Guide" class="color-hover">African Lion</a>
+      <a href="https://www.nationalgeographic.com/news/2016/08/asia-endangered-world-lion-day-conservation/"
+        target="_blank" title="National Geographic" class="color-hover">
+        Asian Lion</a>
+      <a href=https://www.panthera.org/initiative/project-leonardo target="_blank" title="Project Leonardo"
+        class="color-hover">Lion Conservation</a> </navclass="bigger-links">
+  </header>
+
+  <!-- BIG NEWS SECTION -->
+  <section class="big-news">
+
+    <img src="Gapande lejon tbc.jpg" alt="Lion image">
```

Comment:
You actually don't need to have "image" in the `alt` text. Because it is an image element it will read "image" automatically. So in this case just "lion" would be enough!